hydrolib/core/io/ini/models.py

```diff
@@ -185,8 +175,6 @@ def _convert_section(cls, section: Section) -> Dict:
     def _convert_section_to_dict(cls, value: Section) -> Dict:
```

Comment:
Github won't let me leave a comment there but I notice you have the method _combine_in_dict
and I have something similar with the _node_to_dictionary method in the dimr_parser. Not sure if it's exactly the same though.

Reply:
Good to add this note to #40